docs/DeveloperGuide.md

```diff
@@ -46,7 +46,15 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 ### Storage component
 
-{% include DeveloperGuide/Design/storage.md %}
+**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
+
+<img src="images/StorageClassDiagram.png" width="550" />
+
+The `Storage` component,
+* can save address book data, transaction book data and user preference data in JSON format, and read them back into corresponding objects.
+* inherits from `AddressBookStorage`, `TransactionBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
+* depends on some classes in the `Model` component (because the `Storage` component's job is to save/retrieve objects that belong to the `Model`)
```

Comment:
not sure about `depends on some classes in the `Model` component`. If you are referring to `readTransactionBook(AddressBookModel addressBookModel)` in `Storage.java`, then you may need to include a dependency arrow to model class. You can referenced the [Ui class diagram](https://ay2425s1-cs2103-f13-1.github.io/tp/images/UiClassDiagram.png).